applications/FluidDynamicsApplication/python_scripts/fluid_dynamics_analysis.py

```diff
@@ -15,6 +15,15 @@ class FluidDynamicsAnalysis(AnalysisStage):
     def __init__(self,model,parameters):
         # Deprecation warnings
         solver_settings = parameters["solver_settings"]
+
+        if solver_settings.Has("domain_size") and parameters["problem_data"].Has("domain_size"):
+            warn_msg = '"domain_size" defined twice, using the one in "solver_settings"'
```

Comment:
i would change the text to

"defined both in the proble,_data block and in the solver_settings. "the definition in the solver_settings will be employed"